config/opensearch_dashboards.yml

```diff
@@ -237,3 +237,5 @@
 #data_source.encryption.wrappingKeyName: 'changeme'
 #data_source.encryption.wrappingKeyNamespace: 'changeme'
 #data_source.encryption.wrappingKey: [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+
+#opensearchDashboards.isSurveyAllowed: true
```

Comment:
to be honest i think breaking this down into more objects is more scalable and matches the structure of the project.

for example:
```
opensearchDashboards.survey.enabled: true
```

Reply:
comment addressed here https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3521#issuecomment-1487788462